_includes/icons/selfinstalling.svg

```diff
@@ -0,0 +1 @@
+<svg fill="currentColor" version="1.1" viewBox="0 0 24 24" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="m12 2c-5.523 0-10 4.477-10 10s4.477 10 10 10 10-4.477 10-10-4.477-10-10-10zm0 2c0.44823 0 0.88435 0.046315 1.3125 0.11719l-0.65625 3.9375a4 4 0 0 0-0.65625-0.054688 4 4 0 0 0-0.060547 0 4 4 0 0 0-3.9395 4 4 4 0 0 0 0.67188 2.2188l-3.3262 2.2188c-0.84864-1.2701-1.3457-2.7955-1.3457-4.4375 0-4.418 3.582-8 8-8zm1.5605 0.1543c0.034031 0.0067371 0.067688 0.014317 0.10156 0.021484a8 8 0 0 0-0.10156-0.021484zm0.77344 0.19336c0.034744 0.010586 0.068975 0.022159 0.10352 0.033203a8 8 0 0 0-0.10352-0.033203zm0.19336 0.066406c0.68156 0.22707 1.324 0.53998 1.9102 0.93164l-2.2188 3.3262a4 4 0 0 0-0.95508-0.4668l1.2637-3.791zm4.127 3.1484c0.84864 1.2701 1.3457 2.7955 1.3457 4.4375 0 4.418-3.582 8-8 8-0.44823 0-0.88435-0.046315-1.3125-0.11719l0.65625-3.9375a4 4 0 0 0 0.65625 0.054688 4 4 0 0 0 4-4 4 4 0 0 0-0.67188-2.2188l3.3262-2.2188zm-6.7051 2.4375a2 2 0 0 1 0.001953 0 2 2 0 0 1 0.048828 0 2 2 0 0 1 2 2 2 2 0 0 1-2 2 2 2 0 0 1-2-2 2 2 0 0 1 1.9492-2zm-2.168 5.3281a4 4 0 0 0 0.95508 0.4668l-1.2637 3.791c-0.68155-0.22707-1.324-0.53998-1.9102-0.93164l2.2188-3.3262zm-0.21875 4.291a8 8 0 0 0 0.10352 0.033203c-0.034743-0.010586-0.068975-0.022159-0.10352-0.033203zm0.77539 0.20508a8 8 0 0 0 0.10156 0.021484c-0.034031-0.006737-0.067688-0.014317-0.10156-0.021484z"/></svg>
```

Comment:
I would probably go with install icon from https://remixicon.com/#device. Remember to replace fill value with `currentColor`

Reply:
Cool, thanks, will update shortyl.